app/views/spree/order_mailer/cancel_email.html.haml

```diff
@@ -1,11 +1,30 @@
-%h3
-  = t(".customer_greeting", name: @order.bill_address.firstname)
-%h4
-  = t(".instructions")
-%span.clear
+%table.social.white-bg{:width => "100%"}
+  %table.column{:align => "left"}
```

Comment:
can you please replace the hash rockets?
width: "100%"
align: "left"
and below src: "..."

Reply:
what's the `src` for?

Reply:
you should remove the hash rockets like this
```suggestion
table.social.white-bg{width: "100%"}
  = t(".customer_greeting", name: @order.bill_address.firstname)
  %table.column{align: "left"}
```

same for src a few lines below:

%img.float-left{src: "#{@order.distributor.logo.url(:medium)}"}/

Reply:
Alright!